OpenSim/Examples/CustomActuatorExample/toyLeg_example.cpp

```diff
@@ -65,33 +65,35 @@ int main()
         Inertia linkageInertia = Inertia::cylinderAlongY(linkageDiameter/2.0, linkageLength/2.0);
 
         OpenSim::Body* linkage1 = new OpenSim::Body("linkage1", linkageMass, linkageMassCenter, linkageMass*linkageInertia);
-
-        // Graphical representation
-        linkage1->addDisplayGeometry("cylinder.vtp");
-        //This cylinder.vtp geometry is 1 meter tall, 1 meter diameter.  Scale and shift it to look pretty
-        GeometrySet& geometry = linkage1->updDisplayer()->updGeometrySet();
-        DisplayGeometry& thinCylinder = geometry[0];
-        thinCylinder.setScaleFactors(linkageDimensions);
-        thinCylinder.setTransform(Transform(Vec3(0.0,linkageLength/2.0,0.0)));
-        linkage1->addDisplayGeometry("sphere.vtp");
-        //This sphere.vtp is 1 meter in diameter.  Scale it.
-        geometry[1].setScaleFactors(Vec3(0.1));
         
+        // Graphical representation
+        Cylinder cyl;
+        cyl.set_scale_factors(linkageDimensions);
+        Frame* cyl1Frame = new PhysicalOffsetFrame(*linkage1, Transform(Vec3(0.0, linkageLength / 2.0, 0.0)));
+        cyl1Frame->setName("Cyl1_frame");
+        osimModel.addFrame(cyl1Frame);
```

Comment:
> would be inconsistent with all other model components that get "adopted" using "add"

Oh :cry:. @aseth1 should these Model methods have their names changed to "adopt"? I did that in Simbody recently in a few places and left the "add" methods around as deprecated so old code would still work. Maybe file an issue?